lib/config/admin.ts

```diff
@@ -0,0 +1,20 @@
+import { RenovateConfig, RepoAdminConfig } from './common';
+
+let adminConfig: RepoAdminConfig = {};
+
+const derivedAdminOptions = ['localDir'];
+
+export function setAdminConfig(
+  config: RenovateConfig,
+  adminOptions: string[]
+): void {
+  adminConfig = {};
+  const repoAdminOptions = adminOptions.concat(derivedAdminOptions).sort();
+  for (const option of repoAdminOptions) {
+    adminConfig[option] = config[option];
```

Comment:
```suggestion
    adminConfig[option] = config[option];
    delete config[option];
```

Should we remove it from normal config then? So it's no longer passed through?

Reply:
Only once we're finished :)